### PR TITLE
[v2.2.0] Release PR

### PR DESCRIPTION
**Pull request by aidanhahn**

### CODE REVIEW

File: charts/edge-stack/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 This file documents all notable changes to Edge Stack Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v7.2.3
```
Comment (khussey) [inline charts/edge-stack/CHANGELOG.md:6]: This needs to be `7.3.0`.

File: charts/edge-stack/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: 2.1.2
+appVersion: 2.2.0-rc.0
 description: A Helm chart for Ambassador Edge Stack
 name: edge-stack
-version: 7.2.2
+version: 7.2.3-rc.0
```
Comment (khussey) [inline charts/edge-stack/Chart.yaml:5]: Should be `7.3.0-rc.0`.

**Comment by LukeShu** [comment]
I'm concerned that `generate` is failing.

**Review by kflynn — APPROVED** [review]
This looks good as long as `generate` is happy.

**Pull request merged**